autonomous/autonomous.py

```diff
@@ -28,98 +48,85 @@ class AutoBase(AutonomousStateMachine):
 
     logger: logging.Logger
 
-    waypoints: List[Pose2d]
+    waypoints: List[Waypoint]
+    waypoints_poses: List[Pose2d]
```

Comment:
I think this can be inferred by mypy. If not though, it better belongs in the ctor.
```suggestion
```